app-engine/src/main/java/be/cytomine/appengine/handlers/StorageDataEntry.java

```diff
@@ -2,14 +2,13 @@
 
 import java.io.File;
 
-import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 @Data
-@AllArgsConstructor
 @NoArgsConstructor
-public class StorageDataEntry {
+public class StorageDataEntry
+{
```

Comment:
I don't understand, why do you declare all these constructor while having lombok setup ?

Reply:
I don't like this pattern, it means that calling `new StorageDataEntry(data)` would leave you with an object filled with null fields? 
And then what, every time someone wants to use a field from that he would need to check field for null ? 
What's the use case here ?